parser.php

```diff
@@ -18,30 +18,40 @@
 $phenomenon = $decoded->getPresentWeather(); //WeatherPhenomenon array
 $clouds = $decoded->getClouds(); //CloudLayer array
 $windShearAlerts = $decoded->getWindshearRunways();
+$type = array_key_exists ('type',$_GET);
 
 if ($decoded->isValid() == false) {
     exit('Invalid METAR.');
 }
 
 // Airport, date & time
 $airportName = (isset($airports[$decoded->getIcao()])) ? $airports[$decoded->getIcao()] : $decoded->getIcao();
-print($airportName . ' information ' . $_GET['info'] . ', ' . substr($rawMetar, 7, 4) . ' [UTC]');
+if($type and $_GET['type'] == 'D'){
```

Comment:
Use `&&` rather than `and` for consistency, and use `===` in most of the cases rather `==` unless you meant it for safety reasons.